docs/manual/conditionalbuilds.md

```diff
@@ -91,5 +91,16 @@ macros which is nicer in other situations, e.g.:
 
 Always test for the `with`-condition, not the `without`-counterpart!
 
+## Overrinding Defaults
+
+For distributions it can be useful to overwrite the build conditionals on a global scale. To not interfere with the users ability to overwrite the conditionals on the command line there is an option to overwrite the default value indenpendent on the one chosen in the spec file.
+
+To do this one can define a `%bcon_override_default_NAME` macro as one or zero or use the `%{bcon_override_default NAME VALUE}` macro. Distributions can put the former into a global macro file that is installed during local builds to propagate these changed defaults outside their build system. Using different versions of the macro file allows building the same set of packages in different ways - e.g. against different libraries - without altering all the spec files.
+
+E.g. add this in the macros file to disable support for zstd assuming this is a common conditional in the distribution:
+```
+%bcon_override_default_zstd 0
```

Comment:
%bcond_, not %bcon_. Multiple cases of that in the above section.

Reply:
Yeah, that looked funny but then I compared it to the ones above...